src/ScaleUtil/ScaleUtil.js

```diff
@@ -9,6 +9,8 @@ import scaleBand from 'd3-scale/src/band';
  */
 class ScaleUtil {
 
+  static EPSILON = 0.000001;
```

Comment:
Isnt tjere a built in constant?

Reply:
There is for example `Number.MIN_VALUE`, but using that with a log scale still results in lots of `NaN` values in the DOM, so I switched to this arbitrary low number.